src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return (days * 40);
+  } else {
+    if (days < 7) {
```

Comment:
There's no need for such construction.
You may write `else if {}` or just another `if{} `after the first check. Because if the first check pass and some result will be returned, the function will break immediately.